vmslave-config/ansible/roles/install-packer-build-stack/tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+- name: Check if packer is installed
+  shell: |
+    packer version
```

Comment:
you have only 1 line of shell script, no necessary to use Literal Style(|) (new line indictor)